### ISSUE TITLE
js_run_devserver is missing `"ibazel_live_reload"`

### ISSUE THREAD

**Issue by Aghassi** [aspect-build/rules_js#2025]
https://github.com/aspect-build/rules_js/blob/b0b0781963c09fcb25be4553057bfa3a44e05f8d/js/private/js_run_devserver.bzl#L273

I'm not sure if this is on purpose or by accident but the live reload tag is not added by default, meaning ibazel will hang if someone is trying to monitor live reload as well. Should this be added as a default?

**Comment by alexeagle** [comment]
Yeah I see https://blog.engflow.com/2023/10/16/coding-in-the-fast-lane-with-ibazel/#setting-up-livereload-with-ibazel
I think it's just an accident, want to try adding it?
